mlir/lib/Dialect/SCF/TransformOps/SCFTransformOps.cpp

```diff
@@ -217,6 +217,8 @@ loopScheduling(scf::ForOp forOp,
     return 1;
   };
 
+  auto ubConstant = getConstantIntValue(forOp.getUpperBound());
```

Comment:
Nit: please expand `auto` unless the type is obvious from line-level context or impossible to spell out.

Reply:
Thanks, I changed `auto` to `std::optional<int64_t>`